projects/common/src/utilities/url/url-utilities.ts

```diff
@@ -0,0 +1,26 @@
+import { Dictionary, isEmpty } from "lodash"
+
+export const getQueryParamStringFromObject = (params: Dictionary<string | number>): string => {
```

Comment:
I feel like angular already has this type of logic available to us for reuse - `UrlTree.toString()`, perhaps?

Reply:
+1 to this. I looked this up after our conversation @alok-traceable .

We can avoid the custom logic and replace it with

`new HttpParams({ fromObject: queryParams }).toString()`

Reply:
Done suggested change on the util.